packages/compass-e2e-tests/package.json

```
@@ -6,32 +6,38 @@
   "scripts": {
     "clean": "node -e \"try { fs.rmdirSync('.mongodb', { recursive: true }); } catch (e) {}\" && node -e \"try { fs.rmdirSync('.log', { recursive: true }); } catch (e) {}\"",
     "eslint": "eslint",
-    "prettier": "prettier",
+    "prettier": "rm -f fixtures/*.json && prettier",
```

```suggestion
    "prettier": "rimraf fixtures/*.json && prettier",
```

(`rm` is not part of the Windows default environment)

---

That's supposed to be gone (can do the same by just using prettier's ignore file) but somehow it came back due to squash and merge me having branched off the webdriver port branch..